config.toml

```diff
@@ -8,6 +8,11 @@ theme = 'hive'
   apacheURL = 'https://www.apache.org'
   javaDocUrl = 'https://svn.apache.org/repos/infra/websites/production/hive/content/javadocs'
 
+[params.banner]
+    github = "https://github.com/apache/hive"
+    docker = "https://hub.docker.com/r/apache/hive"
+    jira = "https://issues.apache.org/jira/projects/HIVE/issues/HIVE-26644?filter=allopenissues"
```

Comment:
Should it be https://issues.apache.org/jira/browse/${JIRA_ID}? looks like we link to a wrong jira.

Reply:
Or it's better be `https://issues.apache.org/jira/projects/HIVE/issues`?

Reply:
Thanks @dengzhhu653  for the review!  :) 

Updated the url.